net-im/toxic/toxic-0.8.0-r2.ebuild

```diff
@@ -69,7 +69,7 @@ src_configure() {
 	if ! use notifications; then
 		export DISABLE_DESKTOP_NOTIFY=1
 	fi
-	if ! use qrcode; then
+	if ! use qrpng; then
```

Comment:
Don't you need a dep on `media-libs/libpng:0=` for this?

Reply:
I left a comment about this above in this PR. If it's not sufficient to depend on qrencode to pull libpng, I can explicitly put the atom you cited. Frankly I've never been certain about citing dependencies in this scenario. For example many game packages just have immediate deps on libraries but don't list out the common set of X11 libraries they require, because satisfying the first immediate dependencies implies those other deps will be available, too. It's like that, though I think I've also seen a "never depend on transitive dependencies" being an ideology for ebuilds, too. Which is the way Gentoo prefers, I don't yet know.